minimalproject/Packages/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+    "dependencies": {
+        "com.unity.netcode.gameobjects": "file:../../com.unity.netcode.gameobjects"
```

Comment:
no other packages than the SDK (all other built-in modules such as XR, physics, animation, UI, audio etc are removed)